solidity/random-beacon/contracts/RandomBeaconGovernance.sol

```diff
@@ -568,56 +572,101 @@ contract RandomBeaconGovernance is Ownable {
         // slither-disable-next-line reentrancy-no-eth
         randomBeacon.updateDkgParameters(
             newDkgResultChallengePeriodLength,
-            randomBeacon.dkgResultSubmissionEligibilityDelay()
+            randomBeacon.dkgResultSubmissionTimeout(),
+            randomBeacon.dkgSubmitterPrecedencePeriodLength()
         );
         dkgResultChallengePeriodLengthChangeInitiated = 0;
         newDkgResultChallengePeriodLength = 0;
     }
 
-    /// @notice Begins the DKG result submission eligibility delay update
+    /// @notice Begins the DKG result submission timeout update
     ///         process.
     /// @dev Can be called only by the contract owner.
-    /// @param _newDkgResultSubmissionEligibilityDelay New DKG result submission
+    /// @param _newDkgResultSubmissionTimeout New DKG result submission
```

Comment:
..result submission timeout..

We have 2 more places in this test if you search for "eligibility delay"

Reply:
6ae27026d2c534e5b17b254c048cdd77403b95f4